package.json

```diff
@@ -14,6 +14,7 @@
     "react": "^16.9.0",
     "react-aria-offcanvas": "^1.3.4",
     "react-autocomplete": "^1.8.1",
+    "react-body-backgroundcolor": "^1.2.1",
```

Comment:
Seems like it's [deprecated](https://user-images.githubusercontent.com/3723951/74708072-30d83f00-5224-11ea-967d-e83ff331ee87.png) and old

I think you can solve this by setting the background to the [main `div`](https://github.com/Rohithgilla12/find-a-mentor/blob/4ddd93d4f2b54a89abf59f2295cb52e71916a648/src/components/Me/Me.js#L13) and add the necessary css to make it full height (if it's not already)